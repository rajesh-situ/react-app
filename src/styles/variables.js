import styled from 'styled-components';
import React from 'react';

const styles = styled.div`
/* colors */
--main-header-color: #002960;
--main-shadow-color: #0000001a;
/* fonts */
--font-size-medium: 13px;
`;
export default styles;
export const Container = (props) => (
  <styles>
    {props.children}  {/* eslint-disable-line react/prop-types */}
  </styles>
);
