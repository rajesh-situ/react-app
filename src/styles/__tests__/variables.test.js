import React from 'react';
import renderer from 'react-test-renderer';
import { Container } from '../variables';

describe('Variables component', () => {
  it('should match the snapshot', () => {
    const tree = renderer
      .create(<Container />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
