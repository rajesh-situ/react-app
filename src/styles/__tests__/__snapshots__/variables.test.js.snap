// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Variables component should match the snapshot 1`] = `
<styles>
    
</styles>
`;
